authentication/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+# Create your models here.
+
+class User(AbstractUser):
+  uid = models.CharField(primary_key=True, max_length=64)
+  username = models.CharField(unique=True, blank=True, max_length=64)
+  name=models.CharField(max_length=255)
+  college = models.CharField(max_length=255)
```

Comment:
Should we add the list of colleges now @krashish8 ?

Reply:
I don't think there's any issue.

Reply:
Yup!